doc/man3/SSL_shutdown.pod

```diff
@@ -22,18 +22,26 @@ Whether the operation succeeds or not, the SSL_SENT_SHUTDOWN flag is set and
 a currently open session is considered closed and good and will be kept in the
 session cache for further reuse.
 
-The shutdown procedure consists of 2 steps: the sending of the "close notify"
-shutdown alert and the reception of the peer's "close notify" shutdown
-alert. According to the TLS standard, it is acceptable for an application
-to only send its shutdown alert and then close the underlying connection
-without waiting for the peer's response (this way resources can be saved,
-as the process can already terminate or serve another connection).
-When the underlying connection shall be used for more communications, the
-complete shutdown procedure (bidirectional "close notify" alerts) must be
-performed, so that the peers stay synchronized.
+The shutdown procedure consists of 2 steps: sending of the "close notify"
+shutdown alert, and reception of the peer's "close notify" shutdown alert.
+The order of those 2 steps depends on the application.
+
+It is acceptable for an application to only send its shutdown alert and
+then close the underlying connection without waiting for the peer's response.
+This way resources can be saved, as the process can already terminate or
+serve another connection.
+This should only be done in case you're sure that the other side will not
+send more data, otherwise you risk a truncation attack.
+
+When a client only writes, then calls SSL_shutdown() and closes the underlying
+connection, it might not be able to resume the session because it did not
+process the session ticket of the server.
```

Comment:
This feels like there's something left out, I guess that the server has sent a ticket.
So maybe, "..., even if the server has sent a session ticket to establish a session, the client might not be able to resume the session because it did not receive and process the ticket from the server."